hedera-node/src/main/java/com/hedera/services/ledger/TransferLogic.java

```diff
@@ -65,7 +64,6 @@ public class TransferLogic {
 	public static final List<AccountProperty> TOKEN_TRANSFER_SIDE_EFFECTS = List.of(
 			NUM_POSITIVE_BALANCES,
 			NUM_ASSOCIATIONS,
-			HEAD_TOKEN_NUM,
```

Comment:
we do need this right ?? for auto associations on transfer

Reply:
Only the `TokenRelsLinkManager` changes this now.